README.md

```diff
@@ -42,6 +42,13 @@ Additionally, `ode23s` solver supports
 
 There are also fixed step Runge-Kutta and Rosenbrock solvers available.
 
+# Examples
+The examples directory contain a few notebooks that show how to get started. You can also see them here:
+* [Simple differential equation](http://nbviewer.jupyter.org/github/pjpmarques/ODE.jl/blob/master/examples/Simple_Differential_Equation.ipynb)
+* [Lorenz Attractor](http://nbviewer.jupyter.org/github/pjpmarques/ODE.jl/blob/master/examples/Lorenz_Attractor.ipynb)
+* [Terminal Velocity](http://nbviewer.jupyter.org/github/pjpmarques/ODE.jl/blob/master/examples/Terminal_Velocity.ipynb)
+
```

Comment:
"Checked" as in "run-through with no errors"?

No idea - I doubt that it's easy. It would be a nice feature of IJulia, though, so if it doesn't exist it's worth a feature request there. I imagine something like this:

```
using IJulia
run_all_cells("path/to/notebook.ipynb")
```